au.org.access-nri/model/access-om2/experiment/reproducibility/checksums/1-0-0.json

```diff
@@ -0,0 +1,41 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "title": "Access-OM2 Checksums",
+    "description": "The checksums from the model output of Access-OM2 model run",
```

Comment:
The snowplow folks use `self` to store information like the `schema_version` and `schema` URL

https://github.com/snowplow/iglu-central/blob/master/schemas/com.iab.snowplow/spiders_and_robots/jsonschema/1-0-0#L4-L9

```json
    "self": {
        "vendor": "com.iab.snowplow",
        "name": "spiders_and_robots",
        "format": "jsonschema",
        "version": "1-0-0"
    },
```

Do we want to follow that?

I note there are similar ones for other vendors, but maybe those were just created by snowplow following their convention?

https://github.com/snowplow/iglu-central/blob/master/schemas/com.google.analytics/cookies/jsonschema/1-0-0#L4-L9

Reply:
I think that if there are existing conventions, we should follow them...unless we have a better convention. I'll update it.

Reply:
This is snowplow's self-describing JSONs

https://snowplow.io/blog/introducing-self-describing-jsons/

I guess it depends if we think that is a necessary goal

Reply:
Will resolve this due to https://github.com/ACCESS-NRI/schema/pull/5#issuecomment-1947691339